jib-core/src/main/java/com/google/cloud/tools/jib/image/LayerEntry.java

```diff
@@ -69,6 +75,30 @@ public LayerEntry(
                 ? FilePermissions.DEFAULT_FOLDER_PERMISSIONS
                 : FilePermissions.DEFAULT_FILE_PERMISSIONS
             : permissions;
+    this.lastModifiedTime = Math.max(lastModifiedTime, 1000);
```

Comment:
No need for `Max.max()` after passing 1000 in the `LayerEntry` constructor.

Reply:
It is still needed because calling addEntry you can pass your own value and what I understood if we don't want to enable 0 (or < 0) so think it is better to keep it here and it does not really hurt anyway, wdyt?

Reply:
From the plugin usage perspective, this can be meaningful, but from the API design standpoint, I think this library should not incorporate such a high level processing. I think the API should just do what it is told to do so.

Reply:
Works for me, did the change